src/metatensor/models/experimental/soap_bpnn/train.py

```diff
@@ -82,12 +82,6 @@ def train(
         model_capabilities,
     )
 
-    # Create the model:
-    model = Model(
-        capabilities=model_capabilities,
-        hypers=hypers["model"],
-    )
-
```

Comment:
Could you explain what the issue here is and maybe add a test that this issue will not raise in the future again?

Reply:
Yes, essentially the model was being redefined (probably due to a merge issue) so continuing was being overridden by a new model

Reply:
I've added the test